staging/src/k8s.io/dynamic-resource-allocation/controller/controller.go

```diff
@@ -613,9 +614,11 @@ func (ctrl *controller) allocateClaims(ctx context.Context, claims []*ClaimAlloc
 	logger.V(5).Info("Allocating")
 	ctrl.driver.Allocate(ctx, claimsWithFinalizers, selectedNode)
 
+	metrics.DynamicResourceCreateAttempts.Add(float64(len(claimsWithFinalizers)))
```

Comment:
This is not the code which creates ResourceClaim objects. That code is in https://github.com/kubernetes/kubernetes/tree/master/pkg/controller/resourceclaim... which already has metrics support! Ooops.

Sorry for causing unnecessary work. I completely missed that.